python/ray/autoscaler/README.rst

```diff
@@ -0,0 +1,24 @@
+Automated cluster setup and auto-scaling support (Experimental)
+===============================================================
+
+Quick start
+-----------
+
+First, ensure you have configured your AWS credentials in ``~/.aws/credentials``,
+as described in `the boto docs <http://boto3.readthedocs.io/en/latest/guide/configuration.html>`__.
+
+Then you're ready to go. The ``aws/example.yaml`` cluster config file will
```

Comment:
Can we use the full path `ray/python/ray/autoscaler/aws/example.yaml` everwhere in the documentation? It makes it easier to  find the file.

Reply:
Done